src/main/webapp/WEB-INF/pages/productDetails.jsp

```diff
@@ -2,28 +2,65 @@
 <%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
 <%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
 <%@ taglib prefix="tags" tagdir="/WEB-INF/tags" %>
+<style>
+    <%@ include file="/styles/main.css" %>
+</style>
 
 <jsp:useBean id="product" type="com.es.phoneshop.model.entity.Product" scope="request"/>
 <tags:master pageTitle="Product Details">
+    <p>
+        Cart: ${cart}
+    </p>
+    <p>
+        <a href="${pageContext.servletContext.contextPath}/products">Go back</a>
+    </p>
+    <c:if test="${not empty param.message}">
+        <div class="success">
+                ${param.message}
+        </div>
+    </c:if>
+    <c:if test="${not empty param.error}">
```

Comment:
in case of error customer doesn't see the invalid quantity
can we add this ability? To message or save in input field